09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
 
 Please note that there are a number of further GIS software packages that have a scripting interface but for which there is no dedicated R package that accesses these: gvSig, OpenJump, Orfeo Toolbox and TauDEM.
 
-## A short note on further spatial APIs
+## Other bridges
 
-There are many further interfaces to other spatial software packages aside from the interfaces to Desktop GIS software presented in detail in this chapter.
-This includes interfaces to spatial libraries (section \@ref(gdal)), spatial databases (section \@ref(postgis)) and web mapping services (see chapter \@ref(adv-map)).
-In this section we will only have a short glance at two selected examples.
-Hopefully, this will give an impression of all the spatial power and flexibility that is readily available on our fingertips.
+The focus of this chapter is on R interfaces to Desktop GIS software.
+We emphasize these bridges because dedicated GIS software is well-known and a common 'way in' to understanding spatial data, and also because they provide access to many geoalgorithms.
+It is important to be aware of other R bridges, to geospatial programs which based on command-line and are typically accessed via other programs through application programming interfaces (APIs).
+These 'other bridges' include interfaces to spatial libraries (section \@ref(gdal) covers R interfaces to GDAL), spatial databases (section \@ref(postgis)) and web mapping services (see chapter \@ref(adv-map)).
```

Comment:
no, it covers just one interface to GDAL, not several

Reply:
Fixed.